mteb/tasks/Retrieval/ara/SadeemKeywordRetrieval.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class SadeemKeywordRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+
+    metadata = TaskMetadata(
+        name="SadeemKeywordRetrieval",
+        dataset={
+            "path": "sadeem-ai/sadeem-ar-eval-retrieval-keywords",
+            "revision": "6e8b967b0fc44f4cebb34a8cfab0eb7fd9484783",
+            "name": "test",
+        },
+        reference="https://huggingface.co/datasets/sadeem-ai/sadeem-ar-eval-retrieval-keywords",
+        description="SadeemKeyword: A Benchmark Data Set for Community Keyword-Retrieval Research",
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["ara-Arab"],
+        main_score="ndcg_at_10",
+        date=("2024-01-01", "2024-04-01"),
+        form=["written"],
+        domains=["written"],
+        task_subtypes=["Article retrieval"],
+        license="Not specified",
+        socioeconomic_status="medium",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+            @inproceedings{sadeem-2024-ar-retrieval-keywords,
+            title = "{QMS}um: A New Benchmark for Arabic Keywoord-based Articles Searching.",
+            author = "Abu Bakr Soliman | Saddem-ai"
```

Comment:
Is this correctly formatted?

Reply:
Yes. I need to reformat it. Can I clear it?. If not, any recommended format?

Reply:
I removed it